apps/server/prisma/seed.ts

```diff
@@ -0,0 +1,188 @@
+    where: { id: formTemplate1Id },
+    update: {},
+    create: {
+      id: formTemplate1Id,
+      name: 'Form Template 1',
+      formDocLink: 'https://www.mfa.org/'
+    },
+  });
+
+  const signatureFields = await fetchSignatureFields(formTemplate1Id);
+
+  const departmentLeadershipTeam = await prisma.department.upsert({
+    where: { id: LEADERSHIP_TEAM_UUID },
+    update: {},
+    create: {
+      id: LEADERSHIP_TEAM_UUID,
+      name: 'Leadership Team',
+    },
+  });
+  const positions = [
+    {
+      id: CHIEF_OF_STAFF_UUID,
+      name: 'Chief of Staff',
+      departmentId: departmentLeadershipTeam.id,
+      signatureFields: signatureFields,
+    },
+    {
+      id: CHIEF_FIN_OFFICER_UUID,
+      name: 'Chief Financial Officer',
+      departmentId: departmentLeadershipTeam.id,
+      signatureFields: [],  // assuming no signature fields for this position
+    },
+    {
+      id: AGG_DIR_UUID,
+      name: 'AGG Director',
+      departmentId: departmentLeadershipTeam.id,
+      signatureFields: [],  // assuming no signature fields for this position
+    },
+    {
+      id: CHIEF_LEARNING_ENGAGEMENT_UUID,
+      name: 'Chief of Learning & Community Engagement',
+      departmentId: departmentLeadershipTeam.id,
+      signatureFields: [],  // assuming no signature fields for this position
+    },
+  ];
+  for (const position of positions) {
+    await prisma.position.upsert({
+      where: { id: position.id },
+      update: {},
+      create: {
+        id: position.id,
+        name: position.name,
+        departmentId: position.departmentId,
+        signatureFields: {
+          connect: position.signatureFields.map(sigField => ({ id: sigField.id }))
+        }
+      }
+    });
+  }
+
+  const employees = [
+    {
+      id: uuidv4(),
+      firstName: 'Helen',
+      lastName: 'Miao',
+      email: 'email@gmail.com',
+      positionId: CHIEF_OF_STAFF_UUID,
+    },
+    {
+      id: uuidv4(),
+      firstName: 'Kai',
+      lastName: 'Zheng',
+      email: 'email@gmail.com',
+      positionId: CHIEF_FIN_OFFICER_UUID,
+    },
+    {
+      id: uuidv4(),
+      firstName: 'Iris',
+      lastName: 'Zhang',
+      email: 'email@gmail.com',
+      positionId: AGG_DIR_UUID,
+    },
+    {
+      id: uuidv4(),
+      firstName: 'Anshul',
+      lastName: 'Shirude',
+      email: 'email@gmail.com',
+      positionId: CHIEF_LEARNING_ENGAGEMENT_UUID
+      ,
+    },
+  ];
+
+  for (const empData of employees) {
+    await prisma.employee.upsert({
+      where: { id: empData.id },
+      update: {},
+      create: {
+        id: empData.id,
+        firstName: empData.firstName,
+        lastName: empData.lastName,
+        email: empData.email,
+        position: {
+          connect: { id: empData.positionId }
+        }
+      },
+    });
+  }
+}
+
+async function fetchSignatureFields(formTemplateId: string) {
+  const signatureFieldNames = ['Elvin Cheng', 'Isabelle Papa', 'Angela Weigl'];
```

Comment:
the signature field names represent the names of the literal field on the form, so it might be something like `Manager` or `Department Head` or something generic